hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/block/TestDeletedBlockLog.java

```diff
@@ -265,6 +267,15 @@ private void commitTransactions(
         .collect(Collectors.toList()));
   }
 
+  private void commitTransactions(DatanodeDeletedBlockTransactions
```

Comment:
It can be, but the refactor is meaningless.
Mine and the overload function you lined both calls the real commit function ```deletedBlockLog.commitTransactions()```, the difference is that :
The `DatanodeDeletedBlockTransactions` is more complex, the ```Map<UUID, List<DeletedBlocksTransaction>>``` could have different txns of `txID` for different `UUID`, the original overload only commit the same txns of `txID` for different `UUID`.